Bottom: 2976fa73133e0a1bc27b2680b0772a8ac0bd13b2
Top:    e3b7c717a8f83cb60aa48371ea8ef9b147d6a5a6
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-18 11:00:09 +0300

Refresh of sock-resource.patch

---

diff --git a/ib-sock.c b/ib-sock.c
index 7367020..9bbb379 100644
--- a/ib-sock.c
+++ b/ib-sock.c
@@ -156,8 +156,6 @@ cm_server_handler(struct rdma_cm_id *cmid, struct rdma_cm_event *event)
 		list_add(&sock->is_child, &parent->is_child);
 		spin_unlock(&parent->is_child_lock);
 
-		sock_event_set(parent, POLLIN);
-
 		break;
 	}
 	default:
@@ -391,6 +389,5 @@ struct IB_SOCK *ib_socket_accept(struct IB_SOCK *parent)
 		list_del_init(&sock->is_child);
 	}
 	spin_unlock(&parent->is_child_lock);
-	printk("Accept returned %p\n", sock);
 	return sock;
 }
diff --git a/srv.c b/srv.c
index 6f350e9..e09d9d1 100644
--- a/srv.c
+++ b/srv.c
@@ -6,7 +6,7 @@ static int __init
 srv_init(void)
 {
 	struct IB_SOCK *sock, *sock_child = NULL;
-	unsigned long event = 0;
+	unsigned long event;
 	int ret;
 
 	sock = ib_socket_create();
@@ -19,8 +19,7 @@ srv_init(void)
 
 	/* wait incomming connect */
 	while ((event & POLLIN) == 0) {
-		event = ib_socket_poll(sock);
-		printk("srv got event %lx\n", event);
+	event = ib_socket_poll(sock);
 		/* wait a POLLIN to say new connection ready to */
 		if ((event & POLLERR) != 0)
 			goto out;
